src/api/handlers/webhook.ts

```diff
@@ -552,5 +553,10 @@ export async function editSlashResponse(
     options,
   );
 
-  return result;
+  if (options.messageID) return result;
+
+  const message = await structures.createMessage(
+    result as MessageCreateOptions,
+  );
+  return message;
```

Comment:
Are you sure that `Delete Original Interaction Response` returns the message?

Reply:
Github is confusing u this line if part of `editSlashResponse`. For quick confirmation check th eline numbers

Reply:
Ah lol you're right